Assets/Scripts/WeaponAttach.cs

```diff
@@ -0,0 +1,28 @@
+using UnityEngine;
+
+public class WeaponAttach : MonoBehaviour
+{
+    public Sword sword;
+    public Axe axe;
+    public GameObject swordWeapon;
+    public GameObject axeWeapon;
+    public GameObject weaponHolder;
+    public EquippedWeapon currentWeapon;
+
+
+    private void Update()
+    {
+        if (Input.GetMouseButtonDown(1)) axeWeapon.transform.SetParent(weaponHolder.transform);
+
+        if (swordWeapon.transform.IsChildOf(weaponHolder.transform))
+        {
+            currentWeapon.damage = sword.damage;
+            currentWeapon.attackCooldown = sword.attackCooldown;
+        }
+        else if (axeWeapon.transform.IsChildOf(weaponHolder.transform))
+        {
+            currentWeapon.damage = axe.damage;
+            currentWeapon.attackCooldown = axe.attackCooldown;
+        }
```

Comment:
I think a better solution for this would be creating a new function called `SwitchWeapon()` and in the best case, passing in the weapon and setting the values in a single `WeaponHolder` script, or, if working with what we have right now, defining a method like `SwitchWeapon(int damage, float attackCooldown)`